Assets/Scripts/GpsSensor.cs

```diff
@@ -59,14 +61,16 @@ private void ReportPosition()
         // z+ is north, x+ is east
         double[] GPS = Utilities.metersToGPS(new double[] {
             transform.position.z + _noise * Utilities.GaussianRandom(),
-            transform.position.x + _noise * Utilities.GaussianRandom()},
-            new double[] {initLat, initLon});
+            transform.position.x + _noise * Utilities.GaussianRandom(),
+            transform.position.y + _noise * Utilities.GaussianRandom()},
+            new double[] {initLat, initLon, initAlt});
 
```

Comment:
Can probably just insert something like `double altitude = transform.position.y + _noise * Utilities.GaussianRandom();` here and use it in the report below